src/components/OutputPreview.tsx

```diff
@@ -68,10 +79,13 @@ export const OutputPreview = ({ columnQueries, filteredJson }: Props) => {
 						))}
 					</select>
 				</div>
-				<div className="ml-auto">
+				<div className="ml-auto flex gap-2">
 					<button className="border py-1 px-2" onClick={handleCopyEvent}>
 						Copy as {format.toUpperCase()}
 					</button>
+					<a download='JSONReshaperdata' href={fileLink} className="border py-1 px-2" onClick={handleDownload}>
```

Comment:
Let's have the name be like `json-reshaper.csv` or (`.tsv`) depending on the format

Reply:
okay, sir